pool/src/lib.rs

```diff
@@ -618,8 +760,14 @@ impl LendingPool {
         assert_with_error!(env, flags.is_active, Error::NoActiveReserve);
         assert_with_error!(env, amount <= balance, Error::NotEnoughAvailableUserBalance);
 
-        match Self::is_good_position(env, who, None, None, true) {
-            Ok(good_position) => assert_with_error!(env, good_position, Error::BadPosition),
+        let reserves = read_reserves(env);
+        // TODO: fix calc_account_data with balance after withdraw
+        let mb_account_data =
+            Self::calc_account_data(env, who, None, user_config, &reserves, false);
+        match mb_account_data {
+            Ok(account_data) => {
+                assert_with_error!(env, account_data.is_good_position(), Error::BadPosition)
```

Comment:
Can we replace it with the `Self::require_good_position(account_data).unwrap()` here?